docs/gateways/OnDemand.md

```diff
@@ -64,36 +64,44 @@ slurm/<cluster>` with either Alpine or Blanca.
 
 #### Interactive Applications Menu
 
-The _Interactive Applications_ menu contains options to launch certain applications that have graphical user interfaces (GUIs) for interactive use on CURC clusters. Current supported applications include a __remote desktop atop the core cluster__, __MATLAB__, and __JupyterHub__ .
+The _Interactive Applications_ menu contains options to launch certain applications that have graphical user interfaces (GUIs) for interactive use on CURC clusters. Currently supported applications include a __remote desktop__, __MATLAB__, __Jupyter session__, and __RStudio session__.
 
-##### Core Desktop (Remote Desktop)
+##### Core Desktop (remote desktop)
 
-1. When starting a Remote Desktop session, you may customize the resources allocated to the session and other characteristics of the dispatched Slurm job. The default will open a Remote Desktop with 1 node (2 cores) for 1 hour (no need to specify "Account").
-![](OnDemand/core_launch.png)
-2. Click “Launch” to submit the Remote Destop job to the queue. The wait time depends on the number of other users presently on the resource. Requesting smaller, shorter jobs may faciliate shorter wait times. 
-3. When your Remote Desktop is ready, you can click the "Launch Core Desktop" button. In most cases, the default compression and image quality will suffice. If you do have problems with image quality you can adjust these settings as necessary. 
-4. With the Remote Desktop session running and open, you should be able to run standard Linux desktop applications that have a graphical user interface (GUI). 
+1. To start a remote desktop session, you can select either "Core Desktop" or "Core Desktop (Presets)" from the interactive applications menu. When starting a "Core Desktop" session, you may customize the runtime and number of cores for the session. If you select "Core Desktop (Presets)", you may select from standard configurations we provide.
+![](OnDemand/core_desktop_presets.png)
+2. Once either option is selected, click “Launch” to submit the remote desktop job to the queue. The wait time depends on the number of other users presently on the resource. Requesting smaller and shorter jobs may facilitate shorter wait times. 
+3. When your remote desktop is ready, you can click the "Launch Core Desktop" or "Launch Core Desktop (Presets)" button to bring up a web page with the remote desktop. In most cases, the default compression and image quality will suffice. If you do have problems with image quality you can adjust these settings as necessary. 
+![](OnDemand/core_desktop_presets_launch.png)
+4. With the remote desktop session running and open, you should be able to run standard Linux desktop applications that have a graphical user interface (GUI). 
```

Comment:
I think we should change the language from 'should be able to' to 'will be able to' - no biggie, but I think it would be good to have certain language in the docs. Open to debate